hpi-schul-cloud/tldraw-client
# Bc 6342 hide user indicator

# Description

Qa note:
change tested on env:
https://bc-6342-hide-user-indicator.nbc.dbildungscloud.dev/
No issues found . all expected result done.  videos, images upload without any problem.
see: https://ticketsystem.dbildungscloud.de/browse/BC-6342



## Links to Tickets or other pull requests


- https://ticketsystem.dbildungscloud.de/browse/BC-6342


## Changes



## Datasecurity



## Deployment



## New Repos, NPM pakages or vendor scripts



## Screenshots of UI changes



## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
